src/prommis/uky/uky_flowsheet.py

```diff
@@ -3117,26 +3121,26 @@ def add_costing(m):
         * units.day
     )
 
-    solid_waste = value(
-        units.convert(
+    m.fs.solid_waste = Var(m.fs.time, initialize=0.0245, units=units.ton / units.hr)
+    m.fs.solid_waste_constraint = Constraint(
+        expr=m.fs.solid_waste[0]
+        == units.convert(
             m.fs.leach_filter_cake.flow_mass[0], to_units=units.ton / units.hr
         )
     )
 
-    m.fs.solid_waste = Var(
-        m.fs.time, initialize=solid_waste, units=units.ton / units.hr
-    )  # non-hazardous solid waste
-
     m.fs.precipitate = Var(
         m.fs.time, initialize=0, units=units.ton / units.hr
     )  # non-hazardous precipitate
 
-    dust = value(
-        units.convert(m.fs.roaster.flow_mass_dust[0], to_units=units.ton / units.hr)
-    )
     m.fs.dust_and_volatiles = Var(
-        m.fs.time, initialize=dust, units=units.ton / units.hr
-    )  # dust and volatiles
+        m.fs.time, initialize=9.5e-8, units=units.ton / units.hr
+    )
+    m.fs.dust_and_volatiles_constraint = Constraint(
+        expr=m.fs.dust_and_volatiles[0]
+        == units.convert(m.fs.roaster.flow_mass_dust[0], to_units=units.ton / units.hr)
+    )
+
     m.fs.power = Var(m.fs.time, initialize=14716, units=units.hp)
```

Comment:
Tank mixers aren't explicitly modeled in this flowsheet, so I'm not sure how I would write this constraint since none of the unit models (to my knowledge) have variables for power or work.

Reply:
We add some power variables during costing, although they don't seem to add up to 14176 hp (L_tank_mixers 4.74 hp, RSX_tank_mixers 2.0 hp, CSX_tank_mixers 0.08 hp, reep_tank_mixers 0.61 hp). I could have missed some variables.

Reply:
Ok, I've added the constraint setting it equal to the sum of these variables